src/app/dim-ui/PageWithMenu.m.scss

```diff
@@ -54,6 +54,7 @@
   @include phone-portrait {
     position: static;
     margin-right: 0;
+    padding-right: 0;
```

Comment:
This is actually a minor bug fix for mobile to prevent horizontal scroll.

Reply:
This makes a recent change in `LoadoutBuilder.m.scss ` from PR #5959 obselete, can that be removed as part of this PR?

Reply:
ooh good call out, I'll update that. might just split that and this into another pr

Reply:
https://github.com/DestinyItemManager/DIM/pull/5970